docs/stable/5-secure-transport.md

```diff
@@ -47,12 +47,8 @@ It builds on the [X3DH](https://signal.org/docs/specifications/x3dh/) and [Doubl
     - [Multi-device support](#multi-device-support)
     - [Pairing](#pairing)
     - [Sending messages to a paired group](#sending-messages-to-a-paired-group)
-    - [Account recovery](#account-recovery)
+    - [Account recovery](#account-recovery-1)
```

Comment:
Unrelated change / extra number?

Reply:
This is the way GitHub differentiates between anchors of the same name.

The first instance is `#anchor-name` the nth additional instance is referenced `#anchor-name-n` where `n` is the order in which subsequent anchors are placed in the md document.

I'll move these out to #120